packages/frontend/src/components/SimpleStatistic/index.tsx

```diff
@@ -11,25 +9,18 @@ import {
 } from './SimpleStatistics.styles';
 
 const SimpleStatistic: FC = () => {
-    const { resultsData, explore } = useVisualizationContext();
-    const fieldId = resultsData?.metricQuery.metrics[0];
-    let label: string | undefined;
-    if (fieldId) {
-        const field = explore
-            ? findFieldByIdInExplore(explore, fieldId)
-            : undefined;
-        label = field ? getFieldLabel(field) : friendlyName(fieldId);
-    }
+    const { resultsData, bigNumber, bigNumberLabel } =
+        useVisualizationContext();
+
+    const validData = bigNumber && resultsData?.rows.length && bigNumberLabel;
 
-    const bigNumber = bigNumberConfig(resultsData);
-    const validData = bigNumber && resultsData?.rows.length && label;
     return (
         <>
             {validData ? (
                 <SimpleStatisticsWrapper>
                     <BigNumberContainer>
-                        {label && <BigNumberLabel>{label}</BigNumberLabel>}
```

Comment:
@hamzahc1 you left a comment about this but I think there was a discussion recently with @TuringLovesDeathMetal that the label should actually be above rather than below? @PriPatel not sure if you were involved.

Reply:
I got this from how PowerBI and Looker do it. They have the label below the number. 
https://docs.microsoft.com/en-us/power-bi/visuals/power-bi-visualization-card
https://docs.looker.com/exploring-data/visualizing-query-results/single-value-options

This is separate to the chart title which we still have at the top of the tile. Also separate to any prefix/suffix which would be a part of the number.